### PR TITLE
🎉 Source Shopify: migrate to the `Shopify API Version 2023-07` 

### LINKED ISSUE #28127: shopify connector
### Connector Name

shopify

### Connector Version

0.2.63

### What step the error happened?

Other

### Revelant information

I see that source_shopify source.py currently uses api_version = "2022-10" which needs to be upgraded to 2023-04 or later.
There is a warning that I get stating that calls to deprecated apis are being made and it needs to be updated to version 2023-04 or later by UPDATE DEADLINE: JAN 1, 2024.
From shopify:
The gateway, payment_details, and processing_method fields on the Order resource have been removed.
The orders/transactions.json REST endpoint, and the transactions GraphQL connection provide correct information per transaction set. View change

### Relevant log output

_No response_

### Contribute

- [ ] Yes, I want to contribute

### PR DESCRIPTION
**Pull request by bazarnov**
## What
Resolving: https://github.com/airbytehq/airbyte/issues/28127

## How
Edited stream schemas like:

Removed properties:
- `delivery_category` from each stream where the `order shipping line` is available, as it's no longer available within this property, leaving the `delivery_category` everywhere else.
- `gateway, payment_details, processing_method` from [Order](https://shopify.dev/docs/api/admin-rest/2023-07/resources/order) stream

Add New properties:
- `company, confirmation_number, current_total_additional_fees_set, original_total_additional_fees_set, tax_exempt` to [Order](https://shopify.dev/docs/api/admin-rest/2023-07/resources/order) stream
- `po_number` to the [Order](https://shopify.dev/docs/api/admin-rest/2023-07/resources/order) stream and [DraftOrder](https://shopify.dev/docs/api/admin-rest/2023-07/resources/draftorder) streams
- `total_unsettled_set, payment_id` to [Transaction](https://shopify.dev/docs/api/admin-rest/2023-04/resources/transaction) stream.
- `return` to [Order Refund](https://shopify.dev/docs/api/admin-rest/2023-07/resources/refund)
- `created_at, updated_at` to [FulfillmentOrder](https://shopify.dev/docs/api/admin-rest/2023-07/resources/fulfillmentorder)

## 🚨 User Impact 🚨
@erica-airbyte this is the breaking change, all Cloud Customers should be informed prior to the merge. 
Breaking change doc: https://docs.google.com/document/d/13tMPUGC7aw6lY3VcYQfZu3s34VzF6VvthCo6EarmyGg/edit

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-shopify test report (commit [`07f054a5cb`](https://github.com/airbytehq/airbyte/pull/29361/commits/07f054a5cb12ed5994921990dff1cfa0f6909ea8)) - ❌

⏲️  Total pipeline duration: 24mn42s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-shopify/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ❌        |
| Connector package install                                             | ✅        |
| Build source-shopify docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Acceptance tests                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/baz_source-shopify-bump-to-2023-07/1691763547/07f054a5cb12ed5994921990dff1cfa0f6909ea8/source-shopify/0.7.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:07f054a5cb12ed5994921990dff1cfa0f6909ea8)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-shopify test
```

**Comment by octavia-squidington-iii** [comment]
##  source-shopify test report (commit [`065e008cc0`](https://github.com/airbytehq/airbyte/pull/29361/commits/065e008cc0386694a133ebca5682515fd37431f6)) - ✅

⏲️  Total pipeline duration: 24mn42s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-shopify/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-shopify docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Acceptance tests                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/baz_source-shopify-bump-to-2023-07/1691766528/065e008cc0386694a133ebca5682515fd37431f6/source-shopify/0.7.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:065e008cc0386694a133ebca5682515fd37431f6)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-shopify test
```

**Review by artem1205 — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-shopify test report (commit [`e5bc48e394`](https://github.com/airbytehq/airbyte/pull/29361/commits/e5bc48e394c62e6b3d6e84abd1a145f1d880b108)) - ✅

⏲️  Total pipeline duration: 22mn52s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-shopify/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-shopify docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Acceptance tests                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/baz_source-shopify-bump-to-2023-07/1691777809/e5bc48e394c62e6b3d6e84abd1a145f1d880b108/source-shopify/0.7.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:e5bc48e394c62e6b3d6e84abd1a145f1d880b108)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-shopify test
```

**Comment by octavia-squidington-iii** [comment]
##  source-shopify test report (commit [`4217ef781c`](https://github.com/airbytehq/airbyte/pull/29361/commits/4217ef781cbf101279219c6555376e0118d1d3d5)) - ✅

⏲️  Total pipeline duration: 22mn02s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-shopify/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-shopify docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Acceptance tests                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/baz_source-shopify-bump-to-2023-07/1692008650/4217ef781cbf101279219c6555376e0118d1d3d5/source-shopify/0.7.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:4217ef781cbf101279219c6555376e0118d1d3d5)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-shopify test
```

**Review by bnchrch — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-shopify test report (commit [`9b61f0ea0c`](https://github.com/airbytehq/airbyte/pull/29361/commits/9b61f0ea0ceee488b834ac6833dff3b179e7b830)) - ✅

⏲️  Total pipeline duration: 22mn02s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-shopify/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-shopify docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Acceptance tests                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/baz_source-shopify-bump-to-2023-07/1692117730/9b61f0ea0ceee488b834ac6833dff3b179e7b830/source-shopify/0.7.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:9b61f0ea0ceee488b834ac6833dff3b179e7b830)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-shopify test
```

**Comment by bazarnov** [comment]
@nataliekwong Can I ask you to review the public docs updated for this release? Assuming we will solve the `OAuth2.0` release problems by the time of the release (ETA 28.08.2023).

The rendered versions of the Cloud and OSS docs are here:
- Cloud (in app version): https://github.com/airbytehq/airbyte/blob/0e254021a077e76be89af628aa3c41172dd6e435/docs/integrations/sources/shopify.inapp.md
- OSS regular version: https://github.com/airbytehq/airbyte/blob/0e254021a077e76be89af628aa3c41172dd6e435/docs/integrations/sources/shopify.md

**Comment by octavia-squidington-iii** [comment]
##  source-shopify test report (commit [`0e254021a0`](https://github.com/airbytehq/airbyte/pull/29361/commits/0e254021a077e76be89af628aa3c41172dd6e435)) - ✅

⏲️  Total pipeline duration: 21mn44s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-shopify/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-shopify docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Acceptance tests                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/baz_source-shopify-bump-to-2023-07/1692128405/0e254021a077e76be89af628aa3c41172dd6e435/source-shopify/0.7.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:0e254021a077e76be89af628aa3c41172dd6e435)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-shopify test
```

File: docs/integrations/sources/shopify.inapp.md [hunk 1]
```diff
@@ -1,30 +1,26 @@
 ## Prerequisites
 
-* An Active [Shopify store](https://www.shopify.com) 
-* The Admin API access token of your [Custom App](https://help.shopify.com/en/manual/apps/app-types/custom-apps).
+* An Active [Shopify store](https://www.shopify.com)
 
-:::note
+::: note
+
+If you are already a Cloud user but still use the `API PASSWORD` authentication method (option), please make sure you've switched to the `OAuth2.0`, as far as the `API PASSWORD` is going to be deprecated soon for Cloud Users, but this option remains for `OSS` users.
```
Comment (nataliekwong) [inline docs/integrations/sources/shopify.inapp.md:7]: ", as far as the `API PASSWORD` " should read ", as the `API PASSWORD`"

Comment (nataliekwong) [inline docs/integrations/sources/shopify.inapp.md:7]: Suggestion: If you previously used the `API Password` authentication method, please switch to the `OAuth2.0`, as the `API Password` will be deprecated shortly. If you prefer to continue using `API Password`, the option will remain available in Airbyte Open-Source.

File: docs/integrations/sources/shopify.inapp.md [hunk 2]
```diff
@@ -1,30 +1,26 @@
...
-
-1. Navigate to Settings > App and sales channels > Develop apps > Create an app
-2. Name your new app
-3. Select **Configure Admin API scopes**
-4. Tick all the scopes prefixed with `read_` (e.g. `read_locations`,`read_price_rules`, etc ) and save. See below for the full list of scopes to allow.
-5. Click **Install app** to give this app access to your data. 
-6. Once installed, go to **API Credentials** to copy the **Admin API Access Token**. 
+### Connect using OAuth2.0
+1. Click `Authenticate your Shopify account` to start the autentication.
+2. Click `Install` to install Airbyte application.
```
Comment (nataliekwong) [inline docs/integrations/sources/shopify.inapp.md:15]: Suggestion: 2. Click `Install` to install the Airbyte application.

File: docs/integrations/sources/shopify.inapp.md [hunk 3]
```diff
@@ -1,30 +1,26 @@
...
+3. Log in to your account, if not already.
```
Comment (nataliekwong) [inline docs/integrations/sources/shopify.inapp.md:16]: Suggestion: 3. Log in to your account, if you are not already logged in.

File: docs/integrations/sources/shopify.inapp.md [hunk 4]
```diff
@@ -1,30 +1,26 @@
...
+4. Select the store you want to sync and review the consent.
+5. Click on `Install` to finish the Installation.
+6. Reveiew the `Shop Name` field for the chosen store for a sync.
+7. Set the `Start Date` as the starting point for your data replication.
```
Comment (nataliekwong) [inline docs/integrations/sources/shopify.inapp.md:20]: Suggestion: 7. Set the `Start Date` as the starting point for your data replication. Any data created before this date will not be synced.

Comment (nataliekwong) [reply docs/integrations/sources/shopify.inapp.md:20]: I think this is true given how other connectors work, but if not feel free to modify the language.

File: docs/integrations/sources/shopify.inapp.md [hunk 5]
```diff
@@ -1,30 +1,26 @@
...
+8. Click `Test and Save` to finish the source set up.
+
+You're ready to set up Shopify in Airbyte!
```
Comment (nataliekwong) [inline docs/integrations/sources/shopify.inapp.md:23]: Suggestion: remove this line "You're ready to set up Shopify in Airbyte!"

**Review by nataliekwong — COMMENTED** [review]
Would the user need to set up any Custom App scopes if using the OAuth 2.0?

**Comment by bazarnov** [comment]
> Would the user need to set up any Custom App scopes if using the OAuth 2.0?

Nope, it's going to work for the user without the `Custom Apps` created.

File: docs/integrations/sources/shopify.inapp.md [hunk 6]
```diff
@@ -1,30 +1,26 @@
...
+3. Log in to your account, if you are not already logged in.
+4. Select the store you want to sync and review the consent.
+5. Click on `Install` to finish the Installation.
+6. Reveiew the `Shop Name` field for the chosen store for a sync.
+7. Set the `Start Date` as the starting point for your data replication. Any data created before this date will not be synced.
+8. Click `Test and Save` to finish the source set up.
+
+You're ready to set up Shopify in Airbyte!
 
 ### Scopes Required for Custom App
```
Comment (nataliekwong) [inline docs/integrations/sources/shopify.inapp.md:25]: @bazarnov I think if they aren't setting up custom apps  you can remove this section on the individual scopes.

File: docs/integrations/sources/shopify.inapp.md [hunk 7]
```diff
@@ -1,30 +1,26 @@
...
+1. Click `Authenticate your Shopify account` to start the autentication.
+2. Click `Install` to install the Airbyte application.
+3. Log in to your account, if you are not already logged in.
+4. Select the store you want to sync and review the consent.
+5. Click on `Install` to finish the Installation.
+6. Reveiew the `Shop Name` field for the chosen store for a sync.
+7. Set the `Start Date` as the starting point for your data replication. Any data created before this date will not be synced.
+8. Click `Test and Save` to finish the source set up.
+
+You're ready to set up Shopify in Airbyte!
```
Comment (nataliekwong) [inline docs/integrations/sources/shopify.inapp.md:23]: Suggestion to remove this last line as they have already set it up at this point.

**Review by nataliekwong — APPROVED** [review]
Thanks, approving for the docs changes. Please make the same edits to the OAuth section across both the inapp.md and shopify.md, thanks!

**Comment by bazarnov** [comment]
> Thanks, approving for the docs changes. Please make the same edits to the OAuth section across both the inapp.md and shopify.md, thanks!

Done.

**Comment by octavia-squidington-iii** [comment]
##  source-shopify test report (commit [`99b45c69a5`](https://github.com/airbytehq/airbyte/pull/29361/commits/99b45c69a5bee0854602a1ed21e5f1ef15477eb3)) - ✅

⏲️  Total pipeline duration: 16mn57s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-shopify/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-shopify docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Acceptance tests                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/baz_source-shopify-bump-to-2023-07/1692132052/99b45c69a5bee0854602a1ed21e5f1ef15477eb3/source-shopify/0.7.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:99b45c69a5bee0854602a1ed21e5f1ef15477eb3)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-shopify test
```

**Comment by octavia-squidington-iii** [comment]
##  source-shopify test report (commit [`c34c56280d`](https://github.com/airbytehq/airbyte/pull/29361/commits/c34c56280d1f7c36b235c3ce600e3aa4ac878447)) - ✅

⏲️  Total pipeline duration: 17mn03s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-shopify/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-shopify docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Acceptance tests                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/baz_source-shopify-bump-to-2023-07/1692133201/c34c56280d1f7c36b235c3ce600e3aa4ac878447/source-shopify/0.7.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:c34c56280d1f7c36b235c3ce600e3aa4ac878447)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-shopify test
```

**Comment by octavia-squidington-iii** [comment]
##  source-shopify test report (commit [`fa53ec7b6f`](https://github.com/airbytehq/airbyte/pull/29361/commits/fa53ec7b6f31c431f742cc4c0986838f54c6a764)) - ✅

⏲️  Total pipeline duration: 15mn59s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-shopify/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-shopify docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Acceptance tests                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/baz_source-shopify-bump-to-2023-07/1692134360/fa53ec7b6f31c431f742cc4c0986838f54c6a764/source-shopify/0.7.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:fa53ec7b6f31c431f742cc4c0986838f54c6a764)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-shopify test
```

**Comment by octavia-squidington-iii** [comment]
##  source-shopify test report (commit [`8d82af96fd`](https://github.com/airbytehq/airbyte/pull/29361/commits/8d82af96fd6bfbfb0e45ea1b1a19d25a0b0f2a50)) - ✅

⏲️  Total pipeline duration: 22mn16s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-shopify/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-shopify docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Acceptance tests                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/baz_source-shopify-bump-to-2023-07/1692184422/8d82af96fd6bfbfb0e45ea1b1a19d25a0b0f2a50/source-shopify/0.7.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:8d82af96fd6bfbfb0e45ea1b1a19d25a0b0f2a50)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-shopify test
```

**Comment by octavia-squidington-iii** [comment]
##  source-shopify test report (commit [`305f1540b5`](https://github.com/airbytehq/airbyte/pull/29361/commits/305f1540b54efc23a32904fc94aa3e9eb6ea415b)) - ✅

⏲️  Total pipeline duration: 22mn15s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-shopify/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |
| Code format checks                                                    | ✅        |
| Connector package install                                             | ✅        |
| Build source-shopify docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                            | ✅        |
| Acceptance tests                                                      | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/baz_source-shopify-bump-to-2023-07/1692279009/305f1540b54efc23a32904fc94aa3e9eb6ea415b/source-shopify/0.7.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:305f1540b54efc23a32904fc94aa3e9eb6ea415b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-shopify test
```

**Comment by bazarnov** [comment]
We moved this to `On Hold` until Aug 28th, 2023. This is the final fix we need to merge before certification is over.

**Comment by loris** [comment]
@bazarnov Great work!
> saying that the API password would be deprecated for Cloud users
Can you give us more details here? What is the ETA for deprecation and what will happen then (I assume the option will still be available but marked as deprecated) and you will have a future ETA for removing the method completely?
I'm raising concerns because we have many customers connecting using API password method and it will take a significant amount of time to ask them to migrate.

**Comment by bazarnov** [comment]
> 

This is the Shopify requirement, to have the `OAuth2.0` option only for production use. I doubt we have to keep that for a lifetime, but for some time - yes. Please make sure your Customers have direct access to the store they want to use with Airbyte.  Passing `API password` to 3rd parties is restricted by the security policy of Shopify., just FYI. 

@sherifnada @bmcgilli @oustynova @lazebnyi - please chime in, as the person who knows more about partner account usage.

**Pull request merged**